wallet/how-to/use-non-evm-networks/starknet/create-a-simple-starknet-dapp.md

```diff
@@ -1521,7 +1649,13 @@ Start the dapp and navigate to it in your browser.
 yarn start
 ```
 
--->
+Your dapp should look similar to the following:
+
+<div class="row">
+  <div class="column">
+    <img src={require("../../../assets/starknet-tutorial-transfer-token.png").default} alt="Starknet transfer token" width="400" style={{border: "1px solid #DCDCDC"}} />
```

Comment:
```suggestion
    <img src={require("../../../assets/starknet-tutorial-transfer-token.png").default} alt="Starknet transfer token" width="750" style={{border: "1px solid #DCDCDC"}} />
```

Increase image size to match other images. It might be worth cropping this image slightly, too, for readability.